CMakeLists.txt

```diff
@@ -514,7 +514,7 @@ if(ENABLE_MASON)
   set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} ${LINKER_FLAGS}")
 
   # current mason packages target -D_GLIBCXX_USE_CXX11_ABI=0
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_GLIBCXX_USE_CXX11_ABI=0")
+  add_dependency_defines(-D_GLIBCXX_USE_CXX11_ABI=0)
```

Comment:
On Travis Trusty builds, the compilers all had `GLIBCXX_USE_CXX11_ABI=0` by default. This setting has been flipped in Bionic, so we need to explicitly pass this to pkg-config so that the example builds.

Reply:
I forgot why we had this in the first place. I think this was mason related given that is afaik the only place where we care about ABI of our binaries.

Reply:
Yep, the Mason Boosts are built against the old ABI .